src/Minimization_Monitor/image_gen/ush/MinMon_Plt.sh

```diff
@@ -96,9 +97,9 @@ fi
 #  Specify TANKDIR for this suffix
 #--------------------------------------------------------------------
 if [[ $GLB_AREA -eq 1 ]]; then
-   export TANKDIR=${M_TANKverf}/stats/${MINMON_SUFFIX}
+   TANKDIR=${M_TANKverf}/${MINMON_SUFFIX}
```

Comment:
Should `$TANKDIR` be renamed in this script?  Since the configuration file looks for `TANKDIR`, it's a little confusing.  Maybe `$EXP_STATSDIR` or `$EXP_TANKDIR`?

Reply:
Fair point.  I'll rename it.  Thanks.